components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6363,6 +6427,68 @@ func insertStatementProject(t *testing.T, db *testhelpers.TestDB, statementID uu
 	require.NoError(t, err)
 }
 
+func insertAppliedRule(t *testing.T, db *testhelpers.TestDB, rule storage.Rule) {
+	t.Helper()
+	row := db.QueryRow(`
+		INSERT INTO iam_project_rules (id, project_id, name, type) VALUES ($1, $2, $3, $4) RETURNING db_id;`,
+		rule.ID, rule.ProjectID, rule.Name, rule.Type.String())
+	var dbID string
+	err := row.Scan(&dbID)
+	require.NoError(t, err)
+	for _, c := range rule.Conditions {
+		_, err = db.Exec(`
+			INSERT INTO iam_rule_conditions (rule_db_id, value, attribute, operator) VALUES ($1, $2, $3, $4);`,
+			dbID, pq.Array(c.Value), c.Attribute.String(), c.Operator.String())
+		require.NoError(t, err)
+	}
+}
+
+func insertStagedRule(t *testing.T, db *testhelpers.TestDB, rule storage.Rule) {
+	t.Helper()
+	row := db.QueryRow(`
+		INSERT INTO iam_staged_project_rules (id, project_id, name, type, deleted) VALUES ($1, $2, $3, $4, $5) RETURNING db_id;`,
+		rule.ID, rule.ProjectID, rule.Name, rule.Type.String(), false)
+	var dbID string
+	err := row.Scan(&dbID)
+	require.NoError(t, err)
```

Comment:
[nit] This is fine, but we could also do this in one line: `require.NoError(t, row.Scan(&dbID))`